### PR TITLE
Vendor in px-backup-api new changes

### PR DESCRIPTION
**Pull request by px-kesavan**
Signed-off-by: Kesavan Thiruvenkadasamy <kthiruvenkadasamy@purestorage.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by px-kesavan** [comment]
Job : https://jenkins.pwx.dev.purestorage.com/job/Dev/job/torpedo-dev01/167/console

File: go.mod [hunk 1]
```diff
@@ -31,7 +31,7 @@ require (
 	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
-	github.com/portworx/px-backup-api v1.2.2-0.20220801043531-88280e08adeb
+	github.com/portworx/px-backup-api v1.2.2-0.20220822053657-49308ab319f1
```
Comment (lsrinivas-pure) [inline go.mod:34]: is it the latest one? do we have 2.3 or 2.3?

Comment (px-kesavan) [reply go.mod:34]: This the master of px-backup-api (latest)

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**